vela-templates/backend.cue

```diff
@@ -0,0 +1,22 @@
+#Template: {
```

Comment:
how is this going to be used? Are the cue file supposed to be separate from the defintion.yaml or not? We'd better have a consistent pattern.

Reply:
Not used yet, prepare to use this link in definition